Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -1816,31 +1969,30 @@ def new_papi_property_command_ec(raw_response: dict) -> Tuple[list, list]:
     return entry_context, human_readable
 
 
-# Created by D.S.
+# Created by D.S. [Modified on 2023/02/27, add a few new fields]
```

Comment:
Please remove comment, we can see changes in git

Reply:
We have two individuals working on the same integration and only using the Xsoar's Web based development interface.
We are planning to get the CI/CD pipeline setup within this year. However, before that is ready, we are probably going to keep track of some of our work in this manner.